packages/frontend/src/models/CanonicalBridge.ts

```diff
@@ -0,0 +1,341 @@
+import { Signer, ethers, BigNumber, BigNumberish, utils } from 'ethers'
+import {
+  ArbitrumInbox__factory,
+  ArbitrumInbox,
+  L1ArbitrumDaiGateway__factory,
+  L1ArbitrumDaiGateway,
+  L1OptimismDaiTokenBridge__factory,
+  L1OptimismTokenBridge__factory,
+  L1OptimismTokenBridge,
+  L1PolygonPlasmaBridgeDepositManager__factory,
+  L1PolygonPlasmaBridgeDepositManager,
+  L1PolygonPosRootChainManager__factory,
+  L1PolygonPosRootChainManager,
+  L1XDaiForeignOmniBridge__factory,
+  L1XDaiForeignOmniBridge,
+  L1XDaiPoaBridge__factory,
+  L1XDaiPoaBridge,
+  L1XDaiWETHOmnibridgeRouter__factory,
+  L1XDaiWETHOmnibridgeRouter,
+} from '@hop-protocol/core/contracts'
+import { ChainSlug, CanonicalToken } from '@hop-protocol/sdk'
+import Base, { L1Factory } from '@hop-protocol/sdk/dist/src/Base'
+import { metadata } from '@hop-protocol/sdk/dist/src/config'
```

Comment:
I don't think it should be importing from `dist` like this `@hop-protocol/sdk/dist/src/config` because that makes it fragile. I suggest importing from `@hop-protocol/sdk` or structuring the typescript build so the output dirs are more structured like `@hop-protocol/sdk/config`